plugins/heroku/readme.md

```diff
@@ -19,48 +19,73 @@ plugins:
+
+##### Scaling configuration
+
+| Property | Description | Type |
 |-|-|-|
-| `pipeline` | (required) the pipeline name for your application as it's defined in Heroku | none |
-| `systemCode` | (required) system code for your application as it's defined in Biz Ops | none |
-| `scaling` | (required) configuration for dyno scaling with a quantity and size for each production app | none |
+| `size` | the [Dyno type](https://devcenter.heroku.com/articles/dyno-types) for this dyno, e.g. `standard-1x`. apps in the FT Heroku account can only use [professional tier dynos](https://devcenter.heroku.com/articles/dyno-types#dyno-tiers-and-mixing-dyno-types). | `string` |
+| `quantity` | how many of this dyno to use | `number` |
 
-Here's what your configuration might look like passing in the above options:
+##### Example
 
-```yml
+~~~yml
 options:
-  '@dotcom-tool-kit/heroku':
-    pipeline: 'ft-next-static'
-    systemCode: 'next-static'
-    scaling:
-      ft-next-static-eu:
-        web:
-          size: standard-1x
-          quantity: 1
-```
+  tasks:
+    HerokuProduction:
+      scaling:
+        ft-next-static-eu:
+          web:
+            size: standard-1x
+            quantity: 1
+~~~
 
-## Hooks
 
-| Event | Description | Installed to... |
-|-|-|-|
-| `build:remote` | Compile any assets or code required for your app to run. | `heroku-postbuild` script in `package.json` |
-| `release:remote` | Run any post-release tasks for an app that require a tool-kit task, e.g. upload assets to s3 | `heroku-postbuild` script in `package.json` |
+<!-- hide autogenerated schema docs:
 
-## Tasks
+#### Task options
 
-| Task | Description | Preconfigured hooks |
-|-|-|-|
-| `HerokuProduction` | deploy production app to Heroku via CircleCi | `deploy:production` |
-| `HerokuStaging` | deploy to staging | `deploy:staging` |
-| `HerokuReview` | create and test Heroku review app | `deploy:review` |
-| `HerokuTeardown` | scale down staging once smoke tests have been run | `teardown:staging` |
+| Property           | Type                                                                                                                                                                                                                       |
+| :----------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`scaling`** (\*) | _Object with dynamic keys of type_ `string` _and values of type_ _Object with dynamic keys of type_ `string` _and values of type_ _Object with properties:_<ul><li>`size`: `string`</li><li>`quantity`: `number`</li></ul> |
+
+_(\*) Required._
+
+### `HerokuStaging`
+
+Deploy to the Heroku staging app.
+
+
+### `HerokuReview`
+
+Create and deploy a Heroku review app.
+
+
+### `HerokuTeardown`
+
+Scale down the Heroku staging app once it's no longer needed.
```

Comment:
**issue** the docs for the other tasks also seem to have been hidden

Reply:
oh duh the `postProcessMarkdown` runs on the markdown for all the tasks at once